daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

```diff
@@ -96,6 +96,8 @@ An HTTP 204 (No Content) and empty body is returned if successful.
 
 You can get a record in Redis using the `get` operation. This gets a key that was previously set.
 
+This takes an optional parameter `delete` which is by default `false`, when it is set to `true` this operation will use `GETDEL` operation of Redis i.e., it will return the `value` which was previously set and, delete it.
```

Comment:
```suggestion
This takes an optional parameter `delete`, which is by default `false`. When it is set to `true`, this operation will use the `GETDEL` operation of Redis. For example, it will return the `value` which was previously set and delete it.
```

Should this be under the `delete` header on line 140?